change-notes/1.20/analysis-csharp.md

```diff
@@ -11,6 +11,8 @@
 
 | *@name of query (Query ID)*| *Impact on results*    | *How/why the query has changed*                                  |
 | Off-by-one comparison against container length (cs/index-out-of-bounds) | Fewer false positives | Results have been removed when there are additional guards on the index. |
```

Comment:
This table needs a row of pipes and dashes (e.g. |--------|----------|---------|) beneath the column headings.